### PR TITLE
(BR/US) Added lots of channels

### LINKED ISSUE #9065: Add: KBPX-LD5
### Channel ID

KBPXLD5.us

### Stream URL

https://edge1.lifestreamcdn.com/live/geb/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9066: Add: Gem Shopping Network
### Channel ID

GemShoppingNetwork.us

### Stream URL

https://amg01460-gemshoppingnetw-gem-ono-x662c.amagi.tv/playlist720p.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9068: Add: Global Buddhist Network
### Channel ID

GlobalBuddhistNetwork.us

### Stream URL

https://us.gbnus.com/live/dhamma_high/playlist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9069: Add: God's Learning Channel
### Channel ID

GodsLearningChannel.us

### Stream URL

https://2-fss-1.streamhoster.com/pl_122/200562-1176456-1/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9078: Add: Rocky Hill Government Access - Channel 16
### Channel ID

RockyHillChannel16.us

### Stream URL

https://securestream3.champds.com/hlssstc/RockyHillCTLIVE/RockyHillCTLIVE_HQ.m3u8?pID=34871724-045f3362

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9081: Add: WPTV
### Channel ID

WPTV.us

### Stream URL

https://securestream3.champds.com/hlssstc/WestportCTLIVE/WestportCTLIVE_HQ.m3u8?pID=c90fbb07-25b1f072

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9082: Add: CommunityTV Channel 189
### Channel ID

CommunityTVChannel189.us

### Stream URL

https://reflect-tacm.cablecast.tv/live-3/live/WIFI-1596k-540p/WIFI-1596k-540p.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9083: Add: Wolcott Governmental TV
### Channel ID

WLCT96.us

### Stream URL

https://5a5c57d042315.streamlock.net/live11704001/ngrp:government_all/chunklist_w1415698030_b1937864.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9089: Add: Boas Novas
### Channel ID

BoasNovas.br

### Stream URL

https://cdn.jmvstream.com/w/LVW-9375/LVW9375_6i0wPBCHYc/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9091: Add: Canal do Criador
### Channel ID

CanaldoCriador.br

### Stream URL

https://607d2a1a47b1f.streamlock.net/crur/smil:canaldocriador.smil/chunklist_w1977161248_b2596000_sleng.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9092: Add: Canal Metropolitano de Noticia
### Channel ID

CanalMetropolitanodeNoticias.br

### Stream URL

https://video01.logicahost.com.br/redebemtv/redebemtv/chunklist_w1829767969.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9093: Add: Canal Rural
### Channel ID

CanalRural.br

### Stream URL

https://607d2a1a47b1f.streamlock.net/crur/smil:canalrural.smil/chunklist_w1455032094_b2596000_sleng.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9095: Add: Greensboro TV
### Channel ID

GreensboroTV.us

### Stream URL

https://lin11.isilive.ca/live/greensboro/GTN/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9097: Add: Ind TV USA
### Channel ID

IndTVUSA.us

### Stream URL

https://indtv.secure.footprint.net/egress/bhandler/indtv/streama/chunklist_b489472.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9098: It's Supernatural! Network
### Channel ID

ItsSupernaturalNetwork.us

### Stream URL

https://isntv.mmdlive.lldns.net/isntv/f79b65204e7141c6a5cc74e63cf0dae5/chunklist_b3192000.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9099: Add: Greenbelt TV
### Channel ID

GreenbeltTV.us

### Stream URL

https://greenbeltacctv.secure.footprint.net/egress/bhandler/greenbelt/streama/chunklist_b1475584.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9109: Add: NCCTV
### Channel ID

NCCTV.us

### Stream URL

https://newcastlecoutv.secure.footprint.net/egress/bhandler/newcastlecou/streama/chunklist_b1344512.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9110: Add: WITN
### Channel ID

WITN22.us

### Stream URL

https://edge-f.swagit.com/live/wilmingtonde/smil:std-4x3-1-a/chunklist_b512000.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9111: Add: Dover TV
### Channel ID

DoverTV.us

### Stream URL

https://dover-de.secure.footprint.net/egress/bhandler/doverde/streama/chunklist_b1475584.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9112: Add: DCN
### Channel ID

DCN.us

### Stream URL

https://video.oct.dc.gov/out/u/DCN.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9113: Add: DKN
### Channel ID

DKN.us

### Stream URL

https://video.oct.dc.gov/out/u/DKN.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9114: Add: DCC
### Channel ID

DCC.us

### Stream URL

https://video.oct.dc.gov/out/u/DCC.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9115: Add: Room 120 Live
### Channel ID

DCCouncilHearingRoom120.us

### Stream URL

https://video.oct.dc.gov/out/u/97_12.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9118: Add: Jewelry TV Extra
### Channel ID

JewelryTVExtra.us

### Stream URL

https://cdn3.wowza.com/1/R3ZYNjNvdGVoaDFZ/Q3pENnlF/hls/vf0vvvvq/2728/chunklist.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9119: Add: Jewish Life Television
### Channel ID

JewishLifeTelevision.us

### Stream URL

https://jlt2104.cdn.nextologies.com/ebaed9568f2b8e61/ffaaf4b7e1d0e7a8/ea5f88d4e7c252b2/056b5eafae61b1e6/chunks.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9121: Add: World Harvest TV
### Channel ID

WorldHarvestTV.us

### Stream URL

https://597865f6e4114.streamlock.net/live/leseaorigin.stream/chunklist_w1838529020.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9122: Add: Kool TV
### Channel ID

K21GND1.us

### Stream URL

http://64.91.244.153:1935/roku/roku/chunklist_w2078620659.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9123: Add: Blue Mountain TV
### Channel ID

K21JQD1.us

### Stream URL

https://1189614805.rsc.cdn77.org/hls/bmt.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9124: Add: Canal TV Rio
### Channel ID

CanalTVRio.br

### Stream URL

https://video01.logicahost.com.br/canaltvrio/canaltvrio/chunklist_w660183980.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9125: Add: Central TV
### Channel ID

CentralTV.br

### Stream URL

https://cdn-centraltv-11366.ciclano.io:1443/centraltv-11366/centraltv-11366/chunklist_w968700281.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9126: Add: Gazeta
### Channel ID

TVGazeta.br

### Stream URL

https://stream-01.sv4.dailymotion.com/sec(OZ0ERGOH4oE-_Xr-zBQ5v2oHCyYp8Jk3t60mkU27THO4-UbILtM8OM2-Im-7jILR)/dm/3/x89qors/d/live-3.m3u8#cell=lsv4

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #9127: Add: IMPD TV Worldwide
### Channel ID

IMPDTVWorldwide.br

### Stream URL

https://58a4464faef53.streamlock.net/impd/ngrp:impd_all/chunklist_w1460820792_b596000.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by AzeemHadzrie**
That's a lot of channels!

Closes #9127, closes #9126, closes #9125, closes #9124, closes #9123, closes #9122, closes #9121, closes #9119, closes #9118, closes #9115, closes #9114, closes #9113, closes #9112, closes #9111, closes #9110, closes #9109, closes #9099, closes #9098, closes #9097, closes #9095, closes #9093, closes #9092, closes #9091, closes #9089, closes #9083, closes #9082, closes #9081, closes #9078, closes #9077, closes #9069, closes #9068, closes #9066, closes #9065

### CODE REVIEW

**Review by freearhey — DISMISSED** [review]

File: streams/br.m3u [hunk 1]
```diff
@@ -255,3 +255,19 @@ https://5ad482a77183d.streamlock.net/rodrigotvbrusque.com.br/5d880199c902eb4a1e8
 https://cdn.jmvstream.com/w/LVW-8711/ngrp:LVW8711_tG0F3TEBDL_all/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",TV Marajoara (720p)
 https://video01.kshost.com.br:4443/tv31966/tv31966/playlist.m3u8
+#EXTINF:-1 tvg-id="IMPDTVWorldwide.br" status="",IMPD TV Worldwide
+https://58a4464faef53.streamlock.net/impd/ngrp:impd_all/chunklist_w1460820792_b596000.m3u8
+#EXTINF:-1 tvg-id="TVGazeta.br" status="",Gazeta
+https://stream-01.sv4.dailymotion.com/sec(OZ0ERGOH4oE-_Xr-zBQ5v2oHCyYp8Jk3t60mkU27THO4-UbILtM8OM2-Im-7jILR)/dm/3/x89qors/d/live-3.m3u8#cell=lsv4
```
Comment (UltraHDR) [inline streams/br.m3u:261]: Please replace with a non-expiring link

e.g

http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x8924av

File: streams/br.m3u [hunk 2]
```diff
@@ -255,3 +255,19 @@ https://5ad482a77183d.streamlock.net/rodrigotvbrusque.com.br/5d880199c902eb4a1e8
 https://cdn.jmvstream.com/w/LVW-8711/ngrp:LVW8711_tG0F3TEBDL_all/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",TV Marajoara (720p)
 https://video01.kshost.com.br:4443/tv31966/tv31966/playlist.m3u8
+#EXTINF:-1 tvg-id="IMPDTVWorldwide.br" status="",IMPD TV Worldwide
+https://58a4464faef53.streamlock.net/impd/ngrp:impd_all/chunklist_w1460820792_b596000.m3u8
```
Comment (UltraHDR) [inline streams/br.m3u:259]:
```suggestion
https://58a4464faef53.streamlock.net/impd/ngrp:impd_all/playlist.m3u8
```

File: streams/us.m3u [hunk 3]
```diff
@@ -1905,3 +1905,53 @@ https://2-fss-2.streamhoster.com/pl_120/amlst:203530-1426406/playlist.m3u8
 https://5d80890572342.streamlock.net:1936/live/wstoLive/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="online",YouTV
 https://tv.sikkylab.me/hls/stream.m3u8
+#EXTINF:-1 tvg-id="K21JQD1.us" status="",Blue Mountain TV
+https://1189614805.rsc.cdn77.org/hls/bmt.m3u8
+#EXTINF:-1 tvg-id="K21JND1.us" status="",Kool TV
+http://64.91.244.153:1935/roku/roku/chunklist_w2078620659.m3u8
+#EXTINF:-1 tvg-id="WorldHarvestTV.us" status="",World Harvest TV
+https://597865f6e4114.streamlock.net/live/leseaorigin.stream/chunklist_w1838529020.m3u8
```
Comment (UltraHDR) [inline streams/us.m3u:1913]:
```suggestion
https://597865f6e4114.streamlock.net/live/leseaorigin.stream/playlist.m3u8
```

Please replace all the chunklist_*.m3u8 links in us.m3u8 with playlist.m3u8

Thanks

**Comment by LaneSh4d0w** [comment]
There's Record TV Goias that doesn't work, tested with a Brazilian IP.

**Review by BellezaEmporium — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
yarn.lock ?

**Comment by Alstruit** [comment]
I ran the npx command but I had to reinstall the package again. It should be fixed now.

**Review by Alstruit — APPROVED** [review]

**Review by UltraHDR — APPROVED** [review]

**Pull request merged**